src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int index = Main.getRandom().nextInt(4);
+        switch (index) {
+            case 0: {
+                Figure square = new Square(ColorSupplier.getRandomColor(),
+                                                random.nextInt(10));
+                return square;
```

Comment:
```suggestion
                return new Square(ColorSupplier.getRandomColor(),
                                                random.nextInt(10));
```
the same for other figures